---
layout: page
title: Resume
subtitle: 
---

### **Mingsheng Qi**, Ph.D.		
Research Scientist  
Cell: 217-819-6637  
Email: [tonyqms@gmail.com](mailto:tonyqms@gmail.com)    
1130 Olivaire LN, St. Louis, MO 63132  

### OBJECTIVE  
To obtain a position as a Microbiology Scientist that utilizes my research experiences in Microbiome (2+ years), Plant-Microbe Interactions (7+ yeasrs), Molecular Biology and Genetics (10+ years), my impeccable written and verbal communication skills and my PhD degree in Genetics.  

### TRANSFERRABLE SKILLS  
#### Next Generation Sequencing and omics  
Microbiome analyses using amplicon sequencing, mega-genomics; RNA-seq using Illumina HiSeq and differential gene expression analyses; Genome sequencing using 454 sequencing and genome visualization; Cross omics analyses combining genomics, transcriptomics, metabolomics, and hormone and isotope data;
 
#### High throughput techniques  
96-well yeast two hybrid and 384-well real-time PCR; Plant phenotyping assay handling 1140 plants at the same time; 384-droplet yeast two hybrid using liquid handler; 

#### Plant-Microbe interactions  
 Inoculation, infiltration and injection, disease phenotyping in small and large scales, innate and effector-triggered immune responses analyses;
 
#### Molecular Genetics, Cell Biology and Biochemistry  
Molecular cloning, gene expression analyses, protein-protein interactions including yeast two hybrid, BiFC and CoIP, protein expression and purification, HPLC, LC/MS, enzyme engineering, fluorescence imaging, protoplast preparation and transfection, histochemical staining assays;

#### Genome Editing using the CRISPR-Cas9 system in plants

#### IT Skills  
Data analyses;  
Statistics and visualization using Excel, Adobe Illustrator and R packages;  
Image analyses using FIJI;  
Frequent user and proficient mastery of general software, including Microsoft Office, Adobe Photoshop, Endnote and Mendeley;  
Self-learner of JAVA and Python programming.  

### PROFESSIONAL EXPERIENCES   
**Research Scientist**  
Donald Danforth Plant Science Center  
2018/08-present    
**Sorghum-Microbiome interaction**  
Led the project to identify the beneficial rhizosphere bacteria which protect and promote the sorghum plant growth under the drought stress, using high throughput plant phenotyping facility and next-generation sequencing. Discovered the mechanisms behind the phenotypes, using genomic, physiological and biochemical approaches.  

**Postdoc & Assistant Scientist I**  
Iowa State University  
2011/08-2018/08    
**Molecular Plant –Pathogen Interaction**   
A)	Creatively established bacterial screening systems for phenotypic analyses of pathogenic fungal proteins and discovered several novel secreted candidate effector proteins related to fungal pathogenesis.  
B)	Characterized the host interacting proteins of some important virulence effectors from soybean rust pathogen, Phakopsora pachyrhizi, including a cysteine-rich effector and a fungal protease.  
C)	Functionally characterized responses mediated by the soybean rust resistance gene, Rpp1, using virus-induced gene silencing and transcriptome analyses.  

**Postdoctoral Associate**  
University of Illinois at Urbana & Champaign  
2009/02-2011/08  
**Genome Sequencing and Comparative Genomics**   
Led a project to sequence the genome of Peudomonas syringae pv. glycinea strains, which cause bacterial blight on soybean. Conducted in silico subtractive hybridization-based comparative genomic analyses with 9 sequenced phytopathogenic pseudomonads, identified dozens of specific islands (SIs), and revealed the relationship between the SIs and the host specificity of 10 pseudomonads.  

### EDUCATION  
**Ph.D.**  
Genetics  
Chinese Academy of Sciences, China, 2008 

**B.S.**  
Microbiology  
Nanjing Normal University, China, 2003   

### OTHER INFORMATION  
**US Permanent Resident**  

### SELECTED PUBLICATIONS  
* **Mingsheng Qi** and et al. 2019. Candidate effectors from _Uromyces appendiculatus_, the causal agent of rust on common bean, can be discriminated based on suppression of immune responses. Front. Plant Sci. [DOI: 10.3389/fpls.2019.01182](https://www.frontiersin.org/articles/10.3389/fpls.2019.01182/full). IF: 4.298
* **Qi, Mingsheng**, and et al. 2018. QQS orphan gene and its interactor NF-YC4 reduce susceptibility to pathogens and pests. Plant Biotechnology Journal. [DOI:10.1111/pbi.12961](https://onlinelibrary.wiley.com/doi/full/10.1111/pbi.12961). IF: 7.443
* **Mingsheng Qi**, and et al. 2018. Suppression or activation of immune responses by predicted secreted proteins of the soybean rust pathogen _Phakopsora pachyrhizi_. Mol. Plant Microbe Interact. 31: 163-174. [DOI:10.1094/MPMI-07-17-0173-FI](https://apsjournals.apsnet.org/doi/10.1094/MPMI-07-17-0173-FI). IF: 4.455
* **Mingsheng Qi**, and et al. 2016. A small cysteine-rich protein from the Asian soybean rust fungus, _Phakopsora pachyrhizi_, suppresses plant immunity. PLoS Pathog. 12: e1005827. [DOI:10.1371/journal.ppat.1005827](https://journals.plos.org/plospathogens/article?id=10.1371/journal.ppat.1005827). IF: 9.127 
* Steven A Whitham, **Mingsheng Qi**, Roger W. Innes, Wenbo Ma, Valéria Lopes-Caitar, and Tarek Hewezi. 2016. Molecular Soybean-Pathogen Interactions. Annu. Rev. Phytopathol. 54:19.1–19.26. [DOI:10.1146/annurev-phyto-080615-100156](https://www.annualreviews.org/doi/full/10.1146/annurev-phyto-080615-100156). IF: 11.212
* Jason B Noon, **Mingsheng Qi**, Danielle N. Sill, Usha Muppirala, Sebastian Eves-van den Akker, Thomas R. Maier, Drena Dobbs, Melissa G. Mitchum, Tarek Hewezi, Thomas J. Baum. 2016. A Plasmodium‐like virulence effector of the soybean cyst nematode suppresses plant innate immunity. New Phytologist. 212: 444–460. [DOI:10.1111/nph.14047](https://nph.onlinelibrary.wiley.com/doi/full/10.1111/nph.14047). IF: 7.210
* **Mingsheng Qi**, Dongping Wang, Carl A. Bradley, Youfu Zhao. 2011. Genome sequence analyses of _Pseudomonas savastanoi_ pv. _glycinea_ and subtractive hybridization-based comparative genomics with nine pseudomonads. PLoS ONE. 6: e16451. [DOI:10.1371/journal.pone.0016451](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0016451). IF: 4.351
* **Mingsheng Qi**,  Feng-Jie Sun, Gustavo Caetano-Anollés, Youfu Zhao. 2010. Comparative genomic and phylogenetic analyses reveal the evolution of the core Two-Component Signal Transduction systems in Enterobacteria. J Mol Evol. 70: 167-180. [DOI:10.1007/s00239-009-9318-2](https://link.springer.com/article/10.1007%2Fs00239-009-9318-2). IF: 2.762  
[Full publication list](https://scholar.google.com/citations?user=qEQPHFwAAAAJ&hl=en)  

### PROFESSIONAL REFERENCES  
**Chair & Prof. Thomas J. Baum**  
Department of Plant Pathology and Microbiology  
Iowa State University  
Address: 1344 Advanced Teaching & Research Bldg., Ames, IA 50011  
Tel: +1-515-294-5420  
Email: [tbaum@iastate.edu](mailto:tbaum@iastate.edu)   

**Prof. Steve A. Whitham**  
Department of Plant Pathology and Microbiology  
Iowa State University  
Address: 4203 Advanced Teaching & Research Bldg., Ames, IA 50011  
Tel: +1-515-294-4952  
Email: [swhitham@iastate.edu](swhitham@iastate.edu) 

**Prof. Youfu Zhao**  
Department of Crop Sciences University of Illinois  
Address: 288A Edward R. Madigan Laboratory, 1201 W. Gregory Dr., Urbana, IL 61801  
Tel: +1-217-265-9822  
Email: [zhao888@illinois.edu](zhao888@illinois.edu)
